docs/source/guide/pec-3-options.myst

```diff
@@ -415,3 +415,191 @@ uncertainty of the PEC result. For example, given the raw samples contained in
+    circuit=circuit,
+    ideal_executor=Executor(ideal_execute),
+    noisy_executor=Executor(noisy_execute),
+    pec_kwargs={"num_samples": 500},
+    num_training_circuits=5,
+    fraction_non_clifford=0.2,
+    
+    epsilon0=0.9 * epsilon,  # initial guess for epsilon
+)
+```
+
+Upon completing the optimization loop, {func}`.representations.learning.learn_depolarizing_noise_parameter` returns a flag indicating
+whether the optimizer exited successfully, in addition to the optimized noise strength, which can then be input into 
+{func}`.representations.depolarizing.represent_operation_with_local_depolarizing_noise` to calculate quasiprobability representations of the
+ideal gate in terms of noisy gates.
+
+```{code-cell} ipython3
+:tags: ["skip-execution"]
+
+representations = represent_operation_with_local_depolarizing_noise(
+    operations_to_learn, epsilon_opt
+)
+```
+
+### Learning quasiprobability representations with a biased noise model
+
+In cases where the noise of the backend can be approximated by a combined depolarizing and dephasing noise model with a bias factor, also
+referred to as a biased noise model, `pec.representations.learning.learn_biased_noise_parameters` can be used to learn the noise strength
+`epsilon` and noise bias `eta`  associated to a set of input operations. 
+
+
+The single-qubit biased noise channel is given by:
+
+$$
+\mathcal{D}(\epsilon) = (1 - \epsilon)\mathbb{1}
++ \epsilon\Big(\frac{\eta}{\eta + 1} \mathcal{Z}
++ \frac{1}{3}\frac{1}{\eta + 1}(\mathcal{X} + \mathcal{Y}
++ \mathcal{Z})\Big)
+$$ 
+
+
+For multi-qubit operations, the noise channel used is the tensor product of the local single-qubit channels.
+
+```{code-cell} ipython3
+:tags: ["skip-execution"]
+
+from mitiq.pec.representations.learning import (
+    learn_biased_noise_parameters,
+)
+
+epsilon = 0.05
+eta = 0
+
+# simulate biased noise occurs on the CNOT gates
+def noisy_execute(circuit):
+    circuit_copy = circuit.copy()
+    noise_model = NoiseModel()
+    noise_model.add_all_qubit_quantum_error(depolarizing_error(epsilon, 2), ["cx"])
+    return execute_with_noise(circuit_copy, observable, noise_model)
+
+
+operations_to_learn = qiskit.QuantumCircuit(2)
+operations_to_learn.cx(1, 0)
+
+[success, epsilon_opt, eta_opt] = learn_biased_noise_parameters(
+    operations_to_learn=[operations_to_learn],  # learn rep of CNOT
+    circuit=circuit,
+    ideal_executor=Executor(ideal_execute),
+    noisy_executor=Executor(noisy_execute),
+    pec_kwargs={"num_samples": 500, "random_state": 1},
+    num_training_circuits=5,
+    fraction_non_clifford=0.2,
+    training_random_state=np.random.RandomState(1),
+    epsilon0=1.01 * 0.05,  # initial guess for noise strength
+    eta0= eta + 0.01,  # initial guess for noise bias
+)
+```
+
+Upon completing the optimization loop, {func}`.representations.learning.learn_biased_noise_parameters` returns a flag indicating whether the
```

Comment:
This function link is not working.

Reply:
Fixed once branch was updated from master (including commit adding the function `learn_biased_noise_parameters`).